com.unity.render-pipelines.high-definition/Runtime/Material/SharedRTManager.cs

```diff
@@ -113,6 +116,8 @@ public void InitSharedBuffers(GBufferManager gbufferManager, RenderPipelineSetti
                 // In case of deferred, we must be in sync with NormalBuffer.hlsl and lit.hlsl files and setup the correct buffers
                 m_NormalRT = gbufferManager.GetNormalBuffer(0); // Normal + Roughness
             }
+
+            m_DebugDisplayUAV = RTHandles.Alloc(Vector2.one, TextureXR.slices, colorFormat: GraphicsFormat.R32_UInt, dimension: TextureXR.dimension, enableRandomWrite: true, useDynamicScale: true, name: "DebugDisplayUAV");
```

Comment:
@JulienIgnace-Unity we really need to do a pass on memory (note poking you also for render graph).
Currently we allocate a lot of memory even if we don't used thing, I know render graph can help here. But right now situation is still not good.
For the memory above for example 1) we shouldn't allocated in release or if we don't require debug, 2) we could reuse m_ContactShadowBuffer memory (which shouldn't be allocated if we disble shadow).

We will still merge this PR with this, but we really need to do a better memory management

Reply:
Agreed but the question is: Do we want to spend time now doing this for the current code path are should we "just" wait for Render Graph to be enabled by default? Since we target Render Graph for the end of the month I'd say we wait.